CNI/romana

```diff
@@ -184,6 +187,52 @@ set_up_pod () {
 
 	# Kubernetes expects valid json on output so no one above should echo anything in stdout
 	echo -n $R
+
+	update_policy "$POD" "$NAMESPACE"
+}
+
+# update_policy discovers romana network policies that affect current pod
+# and notifies romana agent to include new pod into policy
+update_policy () {
```

Comment:
So policy_definition is a piece of json, and policy is a set of iptables rules.
Name 'update_policy' here refers policy itself - not policy_definition.